DSP-23515 Provide links for older DSE 5.1 and 6.8 Release Notes

tiagomlalves opened the pull request.

szymon-miezal commented:
@tiagomlalves I think those links to the old release notes page are valuable, how about rebasing and merging them?

tiagomlalves commented:
> @tiagomlalves I think those links to the old release notes page are valuable, how about rebasing and merging them?

@szymon-miezal Thanks for noticing this. I've rebased and reworked the commit message. Would you approve the commit?

szymon-miezal reviewed (approved):
That's a valuable addition!
